nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestMigrationObject.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour AL KOTOB
+ */
+package org.nuxeo.rest.management;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static javax.servlet.http.HttpServletResponse.SC_ACCEPTED;
+import static javax.servlet.http.HttpServletResponse.SC_INTERNAL_SERVER_ERROR;
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import static org.apache.commons.io.FileUtils.readFileToString;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.nuxeo.rest.management.migration.MigrationConstants.MIGRATION_ID;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.json.JSONException;
+import org.junit.Test;
+import org.nuxeo.common.utils.FileUtils;
+import org.nuxeo.ecm.core.io.marshallers.json.JsonAssert;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * @since 11.2
+ */
+@Deploy("org.nuxeo.runtime.migration.tests:OSGI-INF/dummy-migration.xml")
+public class TestMigrationObject extends ManagementBaseTest {
+
+    protected static final String STATUS = "status";
+
+    protected static final String ENTITY_TYPE = "entity-type";
+
+    protected static final String MIGRATION = "migration";
+
+    protected static final String DUMMY_MIGRATION = "dummy-migration";
+
+    protected static final String DUMMY_MULTI_MIGRATION = "dummy-multi-migration";
+
+    protected static final String PATH = "site/management/migration/";
+
+    @Test
+    public void testGet() throws IOException, JSONException {
+        try (CloseableClientResponse response = httpClientRule.get(PATH + DUMMY_MIGRATION)) {
+            assertEquals(SC_OK, response.getStatus());
+            JsonNode node = mapper.readTree(response.getEntityInputStream());
+            JsonAssert json = JsonAssert.on(node.toString());
+            File file = FileUtils.getResourceFileFromContext("json/testGet.json");
+            String expected = readFileToString(file, UTF_8);
+            JSONAssert.assertEquals(expected, json.toString(), false);
```

Comment:
Those 3 lines could be extracted in a dedicated method such as `assertJSONResponse`/`checkJSONResponse`/...